pal/src/CMakeLists.txt

```diff
@@ -116,9 +116,9 @@ set(SOURCES
   exception/seh.cpp
   exception/signal.cpp
   file/directory.cpp
-  file/file.cpp
+  file/pal_file.cpp
```

Comment:
The reason for the changing the file names;

Weirdly, combiner was silently overwriting files with same naming under the PAL.